terraform/modules/private-dns-zone-extend/main.tf

```diff
@@ -0,0 +1,34 @@
+provider "aws" {
```

Comment:
I don't think you need to define this here.

Reply:
removed